src/redux.js

```diff
@@ -106,26 +60,8 @@ function* createSession(action) {
   }
 }
 
-function* signupMailingList(action) {
-  try {
-    yield call(waitFor, getSessionCreatedFromState, true)
-    const req = axios.post(`${API_URL}/mailing-list/signup`, {
-      email: action.payload.email,
-      discord_signup: action.payload.discordSignup,
-    })
-    const response = yield req
-    yield put({ type: actionTypes.SIGNUP_MAILING_LIST_SUCCESS, response })
-  } catch (error) {
-    yield put({ type: actionTypes.SIGNUP_MAILING_LIST_FAILURE, error })
-    console.error(error)
-  }
-}
-
 export const saga = function* () {
-  yield all([
-    takeLatest(actionTypes.CREATE_SESSION, createSession),
-    takeLatest(actionTypes.SIGNUP_MAILING_LIST, signupMailingList),
-  ])
+  yield all([takeLatest(actionTypes.CREATE_SESSION, createSession)])
```

Comment:
Non-blocking:

I think we can just
```suggestion
  yield takeLatest(actionTypes.CREATE_SESSION, createSession)
```